pom.xml

```diff
@@ -166,6 +177,7 @@
 		<dependency>
 			<groupId>io.scif</groupId>
 			<artifactId>scifio</artifactId>
+			<version>0.43.2</version>
```

Comment:
This should be set as `scifio.version` rather than hardcoded here. And really, should be inherited from pom-scijava correctly in the first place, like all the rest.